flow/src/main/java/flow/ActivityFlowSupport.java

```diff
@@ -68,10 +69,15 @@ public NonConfigurationInstance(Flow flow) {
     }
   }
 
-  private static final String FLOW_KEY = ActivityFlowSupport.class.getSimpleName() + "_backstack";
+  public static void setBackstackExtra(Intent intent, Backstack backstack, Parceler parceler) {
+    intent.putExtra(BACKSTACK_KEY, backstack.getParcelable(parceler));
```

Comment:
You're making it impossible to manage more than one flow with this static key. Should we allow (not require) specifying a custom key?